uploader.json

```diff
@@ -678,15 +727,23 @@
             "pattern": []
         },
         {
+            "file": "/etc/()*krb5\\.conf",
+            "pattern": []
+        },
+        {
+            "file": "/etc/krb5\\.conf\\.d/()*.*\\.conf",
+            "pattern": []
+        },
+        {
             "file": "/sys/kernel/mm/ksm/run",
             "pattern": []
         },
         {
-            "file": "/etc/security/()*limits.conf",
+            "file": "/etc/security/()*limits\\.conf",
             "pattern": []
         },
         {
-            "file": "/etc/security/limits.d/()*.*",
+            "file": "/etc/security/limits\\.d/()*.*\\.conf",
```

Comment:
Escaping this directory `.d` bit will cause the uploader to fail to find the directory.  If this spec is meant only for sosreport this is ok, but the spec should be wrapped in a `NoneGroup`.